src/maya-stubs/cmds/__init__.pyi

```diff
@@ -634,7 +637,9 @@ def arrayMapper(
     type: str = ...,
 ) -> List[str]: ...
 def art3dPaintCtx(
-    *args: Any,
+    arg0: str = ...,
+    /,
+    *,
```

Comment:
This pattern seem to emerge a lot, if I rebuild the stubs on 2026.2 locally this goes back to `*args: Any` for some reason

this seems to indicate that `cmds.help("art3dPaintCtx")` returns an actual synopsis

On my end I simply get this
```
>>> cmds.help("art3dPaintCtx")
'Quick help is not available for art3dPaintCtx.  \nPlease use help -doc to see online usage documentation.'
```

If you're getting something different, I wonder if that's a Mac vs Windows sort of thing